prez/content/a01.md

```diff
@@ -44,6 +48,7 @@ implementing custom logic if necessary
 - Prefer feature and attribute based access control over RBAC
```

Comment:
I read it as using ABAC instead of RBAC which may or may not be correct based on context. I think it would be better to remove this sentence and explain it with context during the workshop.